here-naksha-app-service/src/test/java/com/here/naksha/app/service/EventHandlerApiTest.java

```diff
@@ -288,4 +288,76 @@ void tc0162_testUpdateEventHandlerWithMismatchingId() throws Exception {
         .hasStreamIdHeader(streamId)
         .hasJsonBody(expectedErrorResponse);
   }
+
+  @Test
+  void tc0180_testDeleteHandler() throws Exception {
+    // Test API : DELETE /hub/handlers/{handlerId}
+    // Given:
```

Comment:
It would be good to provide some comments (even one-liner) for every Given, When, Then steps for someone new to understand what is happening in each test.